timely_beliefs/tests/test_df_resampling.py

```diff
@@ -411,3 +429,26 @@ def test_downsample_instantaneous(df_instantaneous_8111):
     assert df_resampled_2.event_resolution == downsampled_event_resolution
     # frequency updated
     assert df_resampled_2.event_frequency == downsampled_event_resolution
+
+
+def test_upsample_to_instantaneous(df_4111, test_source_a: BeliefSource):
+    """Test upsampling deterministic beliefs about time slot event to instantaneous events."""
+    df = df_4111
+    df = df.resample_events(timedelta(minutes=0))
+    assert df.event_resolution == timedelta(minutes=0)
+    expected_values = [0, 0, 1000, 1000, 2000, 2000, 3000, 3000]
+    expected_event_starts = [
+        pd.Timestamp("2000-01-03T09:00+00"),
```

Comment:
Shouldn't we expect the `time-slot` where the event is and the previous one to be filled-in? Instead of the following one. Is it a timely-belief convention?

Reply:
I don't follow the rationale. Can you elaborate?